cognee-frontend/src/app/globals.css

```
@@ -15,23 +15,16 @@
 }
 
-body {
-  background: var(--global-background-default);
-}
-
 a {
   color: inherit;
   text-decoration: none;
 }
+
+@import "tailwindcss";
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)